docs/operations.rst

```diff
@@ -209,4 +209,48 @@ help flag:
     $ inspirehep crawler job --help
 
 
+Operations in QA
+==========================
+
+Migrate records in QA
+-----------------------------
+
+Connect to QA builder machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. First of all establish a Kerberos authentication (this can be helpfull:
+    http://linux.web.cern.ch/linux/docs/kerberos-access.shtml )
+
+2. After you have run the kinit command and have successfully authenticated you should connect to the builder machine:
+
+.. code-block:: shell
+
+    $ ssh username@inspire-qa-worker3-build1.cern.ch
+
+3. At this point it's a good idea to initialize a screen so you have something to connect to and reestablish your
+    session if something happens to your connection while working remotely to a machine:
+
+.. code-block:: shell
+
+    $ ssh screen
```

Comment:
you are supposed to be already in the machine here, so you might want to remove the ssh there, and probably show in the prompt that you are in the remote machine, something like:
```
# This will also reconnect no a running session if any
build1$ screen -dR
```